### PR TITLE
Fetch typings as a postinstall step

### PR DESCRIPTION
**Pull request by nevir**

### CODE REVIEW

File: generators/app/templates/template/package.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
   "scripts": {
     "watch": "onchange '**/*.d.ts' -e 'out/**' -- npm run build",
     "build": "typings bundle<%- ambient %> -o out",
-    "test": "ts-node test/test.ts"
+    "test": "ts-node test/test.ts",
+    "postinstall": "typings install"
```
Comment (blakeembrey) [inline generators/app/templates/template/package.json]: Never use `postinstall`, you're making every single person do this. Use `prepublish`.

**Comment by unional** [comment]
Why do you add this? The generator will run `typings install` already.
https://github.com/typings/generator-typings/blob/master/generators/app/index.js#L209

File: generators/app/templates/template/package.json (hunk 1 above)
Comment (nevir) [inline generators/app/templates/template/package.json]: oh, I didn't realize that the `typings` CLI tool actually ran `npm install` on these packages!

**Comment by nevir** [comment]
Wanted it to be smooth for anyone who wants to check out the repo and make changes

**Comment by nevir** [comment]
If `postinstall` is out of the question, then maybe updating the generated README & package w/ another step to fetch 'em

File: generators/app/templates/template/package.json (hunk 1 above)
Comment (nevir) [inline generators/app/templates/template/package.json]: But you're right, switching regardless

Comment (blakeembrey) [inline generators/app/templates/template/package.json]: Yeah, `postinstall` runs any time someone installs the package (which means anyone installing `generator-typings` from NPM will fail if they don't have Typings installed, don't worry - you aren't the first person to overlook this :wink:). If you use `prepublish`, it occurs after `npm install` (legacy reasons) and before publishing to NPM so it's usually ideal.

**Comment by unional** [comment]
I see. Thank!

**Pull request merged**